scipy/special/tests/test_orthogonal_eval.py

```diff
@@ -252,3 +253,11 @@ def test_hermite_domain():
     # Regression test for gh-11091.
     assert np.isnan(orth.eval_hermite(-1, 1.0))
     assert np.isnan(orth.eval_hermitenorm(-1, 1.0))
+
+
+@pytest.mark.parametrize("n", [0, 1, 2])
+@pytest.mark.parametrize("x", [0, 1, np.nan])
+def test_hermite_nan(n, x):
```

Comment:
One more tiny thing: either name the test something like `test_gh_11369` or link to the issue in a comment so we know what it’s a regression test for.

Also we usually don’t put docstrings on tests because they make the pytest outputs too verbose.

Reply:
:+1: Removed the docstring, replaced with reference to the original issue using the same format as in the test above.